### PR TITLE
Added content tooltip to ActionRow, disable authority when mod is sel…

### PR DESCRIPTION
**Pull request by TomaszChmielPagepro**
image
https://tw.auroracreation.com/app/tasks/15643415

### CODE REVIEW

File: src/components/v5/common/ActionFormRow/types.ts [hunk 1]
```diff
@@ -11,5 +11,9 @@ export interface ActionFormRowProps {
   children?:
     | ((props: UseToggleReturnType) => React.ReactNode)
     | React.ReactNode;
-  tooltip?: TooltipProps['tooltipContent'];
+  // tooltip?: TooltipProps['tooltipContent'];
```
Comment (CzarekDryl) [inline src/components/v5/common/ActionFormRow/types.ts:14]: Is it needed?

Comment (KornelKwiatkowski) [inline src/components/v5/common/ActionFormRow/types.ts:14]: if this is commented on purpose, please write a comment

Comment (TomaszChmielPagepro) [reply src/components/v5/common/ActionFormRow/types.ts]: Fixed

Comment (TomaszChmielPagepro) [reply src/components/v5/common/ActionFormRow/types.ts]: Fixed

**Review by krawczykkatarzyna — APPROVED** [review]

**Review by KornelKwiatkowski — APPROVED** [review]

**Review by CzarekDryl — APPROVED** [review]

**Pull request merged**